python/smashbox/utilities/monitoring.py

```diff
@@ -0,0 +1,79 @@
+from smashbox.utilities import reflection, config, os
+import smashbox.utilities
+
+def push_to_local_monitor(metric, value):
+    print metric, value
+
+def commit_to_monitoring(metric,value,timestamp=None):
+    shared = reflection.getSharedObject()
+    if not 'monitoring_points' in shared.keys():
+        shared['monitoring_points'] = []
+
+    # Create monitoring metric point
+    monitoring_point = dict()
+    monitoring_point['metric'] = metric
+    monitoring_point['value'] = value
+    monitoring_point['timestamp'] = timestamp
+
+    # Append metric to shared object
+    monitoring_points = shared['monitoring_points']
+    monitoring_points.append(monitoring_point)
+    shared['monitoring_points'] = monitoring_points
+
+def push_to_monitoring(returncode, total_duration):
```

Comment:
push_to_monitoring(returncode, total_duration): takes as a variable final returncode of smashbox run and total_duration of smashbox test